cmd/dexctl/main.go

```diff
@@ -53,7 +56,10 @@ var (
 func init() {
 	log.EnableTimestamps()
 
-	rootCmd.PersistentFlags().StringVar(&global.dbURL, "db-url", "", "DSN-formatted database connection string")
+	rootCmd.PersistentFlags().StringVar(&global.dbURL, "db-url", "", "DSN-formatted database connection string. --db-url flag is deprecated. Use --base-url and --api-key")
+	rootCmd.PersistentFlags().StringVar(&global.baseURL, "base-url", "", "DSN-formatted dex-overlord base URL")
+	rootCmd.PersistentFlags().StringVar(&global.apiKey, "api-key", "", "API key for Admin API")
+	rootCmd.PersistentFlags().StringVar(&global.rootCA, "root-ca", "", "Location of root CA file.This flag is needed if the server uses custome root CA.")
```

Comment:
some typos here. Maybe just "File containing a root CA for the Admin API. If unset, the system CAs will be used."